raiden/tests/utils/mediation_fees.py

```diff
@@ -118,11 +79,52 @@ def amount_without_fees(self) -> PaymentAmount:
         return PaymentAmount(self.total_amount - sum(self.mediation_fees))
 
 
-def get_initial_payment_for_final_target_amount(
-    final_amount: PaymentAmount, channels: List[NettingChannelState]
+def get_amount_with_fees(
+    amount_without_fees: PaymentWithFeeAmount,
+    balance_in: Balance,
+    balance_out: Balance,
+    schedule_in: FeeScheduleState,
+    schedule_out: FeeScheduleState,
+    receivable_amount: TokenAmount,
+) -> Optional[PaymentWithFeeAmount]:
+    """ Return the amount the transfer requires before fees are deducted.
+
+    This function is also used by the PFS. Therefore the parameters should not be Raiden state
+    objects.
```

Comment:
Please also add that this function can return `None` in case of `UndefinedMediationFee` ... which happens when? When the imbalance penalty arrays have not been updated?